src/rpiplatesrecognition/libs/plate_acquisition/config_file.py

```diff
@@ -0,0 +1,38 @@
+"""
+Parameters responsible for plates extraction
+"""
+class ExtractionConfigParameters():
```

Comment:
Hej, jesli to jest taka prosta klasa przechowująca jedynie jakieś dane, to fajnie by było jakbyś przerobił to na tzw. `dataclass`:
```python
from typing import Tuple
from dataclasses import dataclass

@dataclass
class ConfigParameters:
    alg_choice: int = 1

    img_size: Tuple[int, int] = (800, 600)
    max_areas: int = 2
    ...
```